test/testdata/defaultExclude/bin/foo/foo.go

```diff
@@ -0,0 +1,7 @@
+// Copyright 2018 The Anticycle Authors. All rights reserved.
+// Use of this source code is governed by a GPL-style
+// license that can be found in the LICENSE file.
+
+package foo
+
+import "fmt"
```

Comment:
Inconsistency with other import styles.

```suggestion
import (
  "fmt"
)
```

Reply:
Also in some files you use 2 spaces and in others you use 4.

Reply:
Good catch. It should be 1 tab not 4 spaces. `go fmt` doesn't look inside `testdata` directories.